app/components/user-cards.js

```diff
@@ -3,23 +3,38 @@ import { computed } from '@ember/object';
 
 export default Component.extend({
 
-    select_team:"All Employees",
-
-    teamsl : computed ('members.@each.team',function(){
-        return this.get('members').filterBy('team')
+    selected_option:null,
+    
+    teamslist : computed ('selected_option',function(){
+        if(this.selected_option!='All Employees')
+            return this.get('members').filterBy('team',this.get('selected_option'))
+        else
+            return this.get('members').filterBy('team')
+        
     }) ,
 
     teams: computed ('members.@each.team',function(){
         return this.members.mapBy('team').uniq()
+
     }),
+    
+    init() {
+        this._super(...arguments);
+        this.teams.push('All Employees')
```

Comment:
`teams` is a computed property. So, we should avoid mutating it in other places. You can just add this option inside the computed property itself.